src/frontends/paddle/src/op/grid_sampler.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs grid_sampler(const NodeContext& node) {
+    auto data = node.get_input("X");
+    auto grid = node.get_input("Grid");
+    default_opset::GridSample::Attributes attributes{};
+    
```

Comment:
please correct the code format here as log reports.

Reply:
Done